rust/minimal_counter_dapp/src/declarations/minimal_counter_dapp/index.js

```diff
@@ -0,0 +1,38 @@
+import { Actor, HttpAgent } from "@dfinity/agent";
```

Comment:
The declarations/ directory should be in .gitignore and not added to source control